src/main/java/org/jabref/gui/linkedfile/DeleteFileAction.java

```diff
@@ -98,10 +98,16 @@ public void execute() {
                 Path path = file.get();
                 dialogTitle = Localization.lang("Delete '%0'", path.getFileName().toString());
             } else {
+                LOGGER.warn("Could not find file {}", linkedFile.getLink());
                 dialogService.notify(Localization.lang("Error accessing file '%0'.", linkedFile.getLink()));
+
+                // We can trigger deletion of "all" files, because in this case, there is only one files
```

Comment:
```suggestion
                // We can trigger deletion of "all" files from the entry (no deletion on disk), because in this case, there is only one files
```